style.css

```diff
@@ -698,94 +636,95 @@ i.close.show {
     gap: 20px;
     padding-top: 50px;
   }
+}
 
-  #card1 p {
-    width: 286px;
-    height: 99px;
-    font-family: "Inter", sans-serif;
-    line-height: 25px;
-    color: #fff;
-    flex: none;
-    order: 1;
-    flex-grow: 0;
-  }
+/* popup section */
+.popup {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  padding: 10px;
+  background-color: rgba(193, 199, 208, 0.9);
+  z-index: 10;
+}
 
-  #card1 ul {
-    display: flex;
-    flex-direction: row;
-    gap: 12px;
-    width: 247px;
-    height: 40px;
-    flex: none;
-    order: 2;
-    flex-grow: 0;
-    font-family: "Inter", sans-serif;
-    color: #fff;
-  }
+.popup-container {
+  padding: 20px;
+  width: 90%;
+  position: fixed;
+  top: 49%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: #fff;
+  z-index: 11;
+  max-height: 98vh;
+  border-radius: 10px;
+  margin: 20px auto;
+  margin-bottom: 20px;
+}
```

Comment:
- [ ] The popup is looking wonderful but there seems to be a bit of a discrepancy with the Figma design, let me clarify below: 👇
- Figma design:
  ![image](https://user-images.githubusercontent.com/104968236/233352776-d79d5ffd-a67c-4aac-8b4d-7bc18293a77e.png)
- Your design:
  ![Inkedfirefox_E4EeSRhDPE](https://user-images.githubusercontent.com/104968236/233353288-73ee5bbe-8ba8-4bb4-a821-02170aa1727c.jpg)
   - [ ] As I've highlighted in blue above, please fix the spacing issues by comparing it with the Figma design
   - [ ]  Also make sure to follow the proper font family as specified in the Figma design. (The font for the entire text in the popup section needs to be modified)